### ISSUE TITLE
New cop to check items of %i[...] and %w[...] arrays

### ISSUE THREAD

**Issue by fatkodima** [rubocop-hq/rubocop#8886]
I have made the following mistake many times.  For example, in one of the previously implemented cops I wrote
```ruby
RESTRICT_ON_SEND = %i[:== :equal? :eql?].freeze # note the extra `:` symbols
```
and was wondering why it didn't worked.

This can be extended to `%w[...]` arrays and `"` chars. And probably it could be a good idea to detect
```ruby
%i[foo, bar, baz] # note the extra `,`
```

Same goes for `%I[...]` and `%W[...]` arrays.

This cop should be marked as unsafe.

**Comment by marcandre** [comment]
Seems like a good idea to me.

I'd mark the auto-correction unsafe, but the cop itself is probably safe... If you intend `'x,'` or `:":foo"`, don't use this notation as it's definitely not clear to the reader what's going on.

**Comment by owst** [comment]
This sounds related to `Lint/PercentStringArray` and `Lint/PercentSymbolArray` added back in https://github.com/rubocop-hq/rubocop/pull/3165 - are they not working for you @fatkodima?

**Comment by fatkodima** [comment]
Yes, this is what was needed, thanks 👍 This is one more time proves its usefulness.
When I made that mistake, I debugged and fixed before ran a rubocop, so haven't noticed it already exists 😄
